packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -66,6 +69,22 @@ export interface IPortfolio extends cdk.IResource {
    * @param options Options for the initiate share
    */
   shareWithAccount(accountId: string, options?: PortfolioShareOptions): void;
+
+  /**
+   * Associate portfolio with the given product.
+   * @param product A service catalog produt.
+   */
+  addProduct(product: IProduct): void;
+
+  /**
+   * Associate Tag Options
```

Comment:
Can you explain what these are, and why are they important?

Reply:
added this 
```
  /**
   * Associate Tag Options.
   * A TagOption is a key-value pair managed in AWS Service Catalog.
   * It is not an AWS tag, but serves as a template for creating an AWS tag based on the TagOption.
   */
  addTagOptions(tagOptions: TagOption): void;
```